src/test/java/com/odysseusinc/arachne/portal/service/analysis/heracles/parts/InitHeraclesRendererTest.java

```diff
@@ -0,0 +1,40 @@
+package com.odysseusinc.arachne.portal.service.analysis.heracles.parts;
+
+import com.odysseusinc.arachne.portal.service.analysis.heracles.HeraclesAnalysisKind;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static com.odysseusinc.arachne.portal.service.analysis.heracles.parts.HeraclesTestUtils.prefixWithAt;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@RunWith(MockitoJUnitRunner.class)
+public class InitHeraclesRendererTest {
+
+    private InitHeraclesRenderer initHeraclesRenderer;
+
+    @Before
+    public void setUp() {
+        initHeraclesRenderer = new InitHeraclesRenderer();
+    }
+
+    @Test
+    public void shouldRenderInitHeraclesFragmentAndSubstituteKeywords() {
+
+        final String fragment = initHeraclesRenderer.render(HeraclesAnalysisKind.FULL);
+
+        assertThat(fragment).doesNotContain(prefixWithAt(initHeraclesRenderer.getParameters()));
```

Comment:
tbh I would remove `prefixWithAt` method and just add a check method:
```
checkThatAllParametersWereReplaced() {
assertThat(fragment).doesNotContain(initHeraclesRenderer.getParameters().addPrifex()));
}
```

Reply:
It was in the initial version but I have decided to move it to test because there is no usages of "get Parameters with prefix" in the prod code. I can change the name to something like 'renameToSqlParamer' instead of prefixWithAt